tests/Fixer/Operator/NewWithParenthesesFixerTest.php

```diff
@@ -0,0 +1,869 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\Operator;
+
+use PhpCsFixer\Tests\Test\AbstractFixerTestCase;
+
+/**
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\Fixer\Operator\NewWithParenthesesFixer
+ */
+final class NewWithParenthesesFixerTest extends AbstractFixerTestCase
```

Comment:
potentially we could apply same idea for all deprecations about not-duplicating tests?

(not only this PR)

(not enforcing, up to you)

Reply:
I've disabled auto-merge for now, re-enable it if you really don't want to wait for @localheinz' response 😉.

Reply:
you asked me to not block things. this could be added as separated PR. i go for hols today and want to release this today.